ajax/libs/guards/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "guards",
+  "filename": "guards.min.js",
+  "version": "1.5.1",
+  "title": "guards.js",
+  "author": {
+    "name": "Mike Virata-Stone",
+    "email": "stonemj+jqueryplugins@gmail.com",
+    "url": "http://reasonnumber.com/"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "jquery": "~1.4.4"
+  },
+  "description": "Guard your forms with class (or any other selector).",
+  "keywords": [
+    "validation"
+  ],
+  "homepage": "http://guardsjs.com/",
+  "docs": "http://guardsjs.com/documentation.html",
+  "demo": "http://guardsjs.com/",
+  "download": "http://guardsjs.com/downloads.html",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/on-site/guards.js.git",
+    "basePath": "",
+    "files": [
+      "downloads/guards*"
```

Comment:
@dannyxx001 it's better to take `downloads` as `basePath`, so that we don't need to add that folder.